cpu/k22f/Makefile.dep

```diff
@@ -0,0 +1 @@
+-include $(RIOTCPU)/kinetis_common/Makefile.dep
```

Comment:
I see the logic for these files (and normally I am the one complaining about this). But could we simply assume that the boards know that e.g. the k22f is build around the kinetis_common code and call this include directly from the board, skipping the need of this additional file (and others elsewhere)?

Reply:
In the kinetis-case, it would work. But I must say I'd rather keep the extra file than require everyone that deals with boards know things like "k22f is a kinetis cpu and those don't include the cpu makefile features but kinetis_common one instead"...